projects/plugins/jetpack/_inc/lib/components.php

```diff
@@ -23,10 +23,25 @@ public static function render_component( $name, $props ) {
 		$rtl = is_rtl() ? '.rtl' : '';
 		wp_enqueue_style( 'jetpack-components', plugins_url( "_inc/blocks/components{$rtl}.css", JETPACK__PLUGIN_FILE ), array( 'wp-components' ), JETPACK__VERSION );
 
-		ob_start();
-		// `include` fails gracefully and throws a warning, but doesn't halt execution.
-		include JETPACK__PLUGIN_DIR . "_inc/blocks/$name.html";
-		$markup = ob_get_clean();
+		/*
+		 * When making changes to the markup here,
+		 * you may want to make similar changes to the editor markup,
+		 * generated in projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/index.jsx
+		 */
+		$markup = '';
+		if ( 'frontend-nudge' === $name ) {
```

Comment:
We could consider using [`get_template_part`](https://developer.wordpress.org/reference/functions/get_template_part/) here, since it now supports `$args`. It would make it easier to edit the markup of the individual components.

That being said, maybe this isn't necessary until (and if) we add more involved components.

Reply:
I think it would be worth it if the content were more than just a few lines. Until then, I think we can keep this inline like that.

Reply:
Yeah, fully agree with that.